servicemesh-install/topics/tutorial-kiali.adoc

```diff
@@ -37,61 +37,101 @@ tracing                tracing-istio-system.127.0.0.1.nip.io
 
 +
 . Launch a browser and navigate to https://${KIALI_URL} (in the output above, this is `kiali-istio-system.127.0.0.1.nip.io`).  You should see the Kiali console login screen.
-+
-image:kiali-login.png[Login Page]
-+
-Log in to the Kiali console using the user name and password that you specified in the custom resource file during installation.
+
+. Log in to the Kiali console using the user name and password that you specified in the custom resource file during installation.
```

Comment:
In the GA, this depends on the auth.strategy you set. If you set it to "login" then this is true. If "openshift", then you log into Kiali using your openshift user credentials. If "anonymous" there is no login credentials required

Reply:
@jmazzitelli, we're still talking TP11 on this PR.  I'll make a note about this in my GA drafts.  Thanks!

Reply:
And I think in TP12 / GA this should be always openshift for the product.
Then "login" option should be seen more an option for community scenarios rather than product. I think.
cc @mwringe